protostar/starknet/cheatable_execute_entry_point.py

```diff
@@ -54,9 +54,12 @@
 logger = logging.getLogger(__name__)
 
 
+ContractFilename = str
+
+
 @dataclass(frozen=True)
 class ContractProfile:
-    callstack: List[str]
+    callstack: List[ContractFilename]
```

Comment:
I find it confusing because `Sample::callstack` has a different type (`List[Instruction]`).

Reply:
Because there are two different callstacks, contract callstack(cairo functions) and transaction callstack(called contracts). I will rename it to transaction_callstack